examples/data-sources/bloxone_dns_delegations/data-source.tf

```diff
@@ -0,0 +1,23 @@
+# Get delegation zone filtered by an attribute
+data "bloxone_dns_views" "example_by_attribute" {
+  filters = {
+    "name" = "example_view"
+  }
+}
+
+data "bloxone_dns_delegations" "example_by_attribute" {
+  filters = {
+    fqdn = "tf-acc-test.com."
+    view = data.bloxone_dns_views.example_by_attribute.results.0.id
```

Comment:
do we really need `view` to filter by fqdn, if not we should avoid it in this example